front/external.php

```diff
@@ -30,6 +30,8 @@
 
 include('../../../inc/includes.php');
 
+Session::checkLoginUser();
+
```

Comment:
Since alerts can be displayed from the login page, we only need to check whether the user is logged in or not.

This is already handled a few lines below with:

```php
$logged = isset($_SESSION['glpiID']);
PluginNewsAlert::displayAlerts(['show_only_login_alerts' => !$logged]);
```

Therefore, no additional check is necessary here.